src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -213,16 +213,15 @@ public static class Diagnostics {
 			defaultSeverity: DiagnosticSeverity.Error,
 			isEnabledByDefault: true,
 			description: "The default constructor for ImmutableArray<T> doesn't correctly initialize the object and leads to runtime errors. Use ImmutableArray<T>.Empty for empty arrays, ImmutableArray.Create() for simple cases and ImmutableArray.Builder<T> for more complicated cases."
-		);
-
+		);
+
 		public static readonly DiagnosticDescriptor UnnecessaryMutabilityAnnotation = new DiagnosticDescriptor(
 			id: "D2L0030",
```

Comment:
D2L0030 is currently unused. Looks like https://github.com/Brightspace/D2L.CodeStyle/pull/325 was ramping up to using it, but it never got there.